### PR TITLE
allow for listing of FAQs

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
_tap_ _tap_ is this thing on?
...
READY FOR REVIEW!!

File: go-app-browser.js [hunk 1]
```diff
@@ -134,7 +150,45 @@ go.app = function() {
...
+                        return error;
+                    } else {
+                        return response.data.map(function (d) {
+                            return new Choice(d.id, d.title);
+                        });
+                    }
+                })
+                .then(function (choices) {
+                    return new PaginatedChoiceState(name, {
+                        question: $('Welcome to FAQ Browser. Choose FAQ:'),
```
Comment (imsickofmaps) [inline go-app-browser.js:175]: Given this state is now like a welcome of welcomes it might be confusing to have to screens saying welcome

Comment (imsickofmaps) [inline go-app-browser.js:175]: Ach. Commenting on complied, not source. Sorry.

Comment (smn) [inline go-app-browser.js:175]: Yeah, not ideal, what would you suggest? A config switch or leave it as is and leave it for `i18n` to update?

**Comment by imsickofmaps** [comment]
:+1: ... just wish there was a better way to have an "English configurable welcome, that can also be translated"

**Comment by smn** [comment]
OK, separate PR for that one then?

**Comment by imsickofmaps** [comment]
Yes, if that's even possible?

**Comment by imsickofmaps** [comment]
Logged in #10

**Pull request merged**